scripts/release-branch.js

```diff
@@ -0,0 +1,146 @@
+
+// GitHub API
+const https = require('https');
+
+// Added simple-git to use for git add "npm install simple-git"
+// Could not use the shell.exec to git add the remove changelogs.
+const git = require('simple-git');
+
+// // Find out the latest release tag and display it in the command line.
+// const latest = shell.exec('git describe --abbrev=0 --tags');
+//
+// // Display the latest tag.
+// console.log(`Display the latest tag: ${latest}`);
+//
+// // Increment the release branch.
+// const minor = semver.inc(latest.toString(), 'minor');
+//
+// // Print out the new minor version
+// console.log(`New release tag: ${minor}`);
+//
+// // Update the changelog.md file
+// const today = new Date();
+// const day = today.getDay();
+// const month = today.getMonth();
+// const year = today.getFullYear();
+//
+// // Changelog.md title for each release
+// const title = `## ${minor} (${month}/${day}/${year})`;
+// // Look at the changelog files
+//
+// const directoryPath = path.resolve(__dirname, '../changelogs');
+// const changelogPath = `${path.resolve(__dirname, '../')}/CHANGELOG.md`;
+//
+// let newLogs = [];
+// // Read directory path and exclude the template.yml file.
+// const changelogs = fs.readdirSync(directoryPath).filter(function(template) {
+//   return template !== 'template.yml';
+// });
+//
+// changelogs.forEach((fileName) => {
+//   const content = yaml.safeLoad(fs.readFileSync(`${directoryPath}/${fileName}`, 'utf8'));
+//   Object.keys(content).forEach((changeType, i) => {
+//     newLogs.push(`\n### ${changeType} \n`)
+//     content[changeType].forEach((change) => {
+//       const newChange = `- (${change.project}) [${change.component}] ${change.issue}: ${change.description}\n`
+//       newLogs.push(newChange);
+//     });
+//   });
+// });
+//
+// const fd = fs.readFileSync(changelogPath).toString().split("\n");
+// fd.splice(0, 0, title, newLogs.join(''));
+// var allLogs = fd.join('\n');
+//
+// // Remove the changelog files
+// fs.readdir(directoryPath, function(err, items) {
+//   for (var i=0; i<items.length; i++) {
+//     if (items[i] != "template.yml") {
+//       var changeLogFilePath = directoryPath + "/" + items[i];
+//       fs.unlink(changeLogFilePath, (err) => {
+//         if (err) throw err;
+//       });
+//     }
+//   }
+// });
+//
+// fs.writeFileSync(changelogPath, allLogs, (err) => {
+//   if (err) throw err;
+// })
+//
+// // Checkout the branch.
+// const releaseBranch = 'release/' + minor;
+//
+// git().checkoutLocalBranch(releaseBranch, () => {
+//         console.log(`On current release branch: ${releaseBranch}`)
+//       })
+//      .add('./*')
+//      .commit('changelog update and remove old changelog files')
+//      .push('origin', releaseBranch);
+
+
+
+console.log(process.env.DANGER_GITHUB_API_TOKEN)
+
+const owner = 'massgov';
+const repo = 'mayflower';
+const title = 'Release/test';
+const head = 'release/9.34.0';
+const base = 'master';
+
+octokit.pulls.create({
```

Comment:
This returns a promise. You can either deal with this using `.then()`, or wrap it inside of an `async` function.  For simplicity, let's just go with this:
```
octokit.pulls
  .create({ ... })
  .catch(function() {
    console.error(`There was an error creating the Github PR: ${e.toString()`);
    process.exit(1);
  })
```
Because this is a promise, anything written after this line may be executed _before_ the PR finishes creating, which is a bit confusing.  So either make this the last thing that happens in the script, or put the rest of the stuff that needs to happen inside a `.then()` callback chained off the same function.